cc-env_test.go

```diff
@@ -432,6 +432,74 @@ func TestCCEnvGetEnvInfo(t *testing.T) {
 	assert.Equal(t, expectedCCEnv, ccEnv)
 }
 
+func TestCCEnvGetEnvInfoNoHypervisorVersion(t *testing.T) {
+	assert := assert.New(t)
+
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		panic(err)
+	}
+	defer os.RemoveAll(tmpdir)
+
+	const logFile = "/tmp/file.log"
```

Comment:
I see a few of these, and it feels like maybe this is wired somewhere else as well - is this something we can lift out to a single place? (but tbh, I fully expect you would have done that if it were possible).

Reply:
We probably could get rid of some of those. I do think we could do with another test sprint since although our coverage level is high, there is still quite a bit of refactoring we could do tbh.